priv/perf/apps/omg_load_testing/test/load_tests/childchain_test.exs

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule ChildChainTest do
+  @moduledoc """
+  childchain load test
+  """
+  use ExUnit.Case
+
+  alias OMG.LoadTesting.Utils.Faucet
+  alias OMG.LoadTesting.Utils.NonceTracker
+
+  @tag timeout: 6_000_000
+  test "childchain test" do
```

Comment:
this seems very random to start the processes like this... is there a better way maybe?